float32 Sx
float32 Sy
float32 Syaw
---
visualization_msgs/Marker Left_path
visualization_msgs/Marker Right_path
